.gitignore

```diff
@@ -2,6 +2,8 @@
 /node_modules
 /Tests/env/local
 /vendor
+/classes/Dependencies
```

Comment:
The namespaced dependencies should be versioned, so that they are available when we publish the release via the Github action